client_plugin/utils/powershell/powershell.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A thread-safe PowerShell utility that reuses a single session.
+
+// +build windows
+
+package powershell
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	ps "github.com/gorillalabs/go-powershell"
```

Comment:
Mark has approved use of that package. The package is very small in itself, so it shouldn't be of too much concern.